### ISSUE TITLE
Lack of any progress information

### ISSUE THREAD

**Issue by netmindz** [vog/bscp#13]
As this is designed for copying block devices, it's highly likely that we are talking about multiple gigabytes of data, so there should really be some form of progress information

**Comment by vog** [comment]
Indeed, this would be a nice feature. There is even a pull request for that:

* #5 Progress Display

Unfortunately, this PR had some issues which the PR's author didn't fix (yet). Any help and/or new pull request is welcome!

**Comment by vog** [comment]
To anyone involved in this issue, please see #16 "Future of bscp".

**Comment by vog** [comment]
> Indeed, this would be a nice feature. There is even a pull request for that:
> 
>     * [Progress Display #5](https://github.com/bscp-tool/bscp/pull/5) Progress Display
> 
> 
> Unfortunately, this PR had some issues which the PR's author didn't fix (yet). Any help and/or new pull request is welcome!

Alas, I finally had to close and reject that PR, due to the outstanding problems.

A usable pull request for showing progress information is still very welcome.
